index.html

```diff
@@ -0,0 +1,191 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Youtube-Clone</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+
+<body>
+    <nav>
+        <div class="left-icons">
```

Comment:
Great on implementing list items in a nav, but it's better to use semantic unorder list ul for all the listed items inside the nav for better code readability, user experience, and greater accessibility. Kindly use a ul tag instead of a nav.